### PR TITLE
Add environment variable allowing patterns to be specified

### PR DESCRIPTION
**Pull request by klgilbert**
The image just needs to be rebuilt and updated in the registry.

### CODE REVIEW

File: rails/rails_releases/rails_benchmarks/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM rubybench/ruby:0.2
+FROM rubybench/ruby:0.3
```
Comment (tgxworld) [inline rails/rails_releases/rails_benchmarks/Dockerfile]: Why is this changed here?

File: rails/rails_releases/rails_benchmarks/runner [hunk 2]
```diff
@@ -28,4 +32,4 @@ bundle install -j4
 
 echo "Running benchmarks with Rails $RAILS_VERSION"
 cd /ruby-bench-suite/rails/benchmarks
-ruby driver.rb -r 2
+ruby driver.rb -r 2 $PATTERNS
```
Comment (tgxworld) [inline rails/rails_releases/rails_benchmarks/runner:35]: Lets update the README to reflect this change :smile_cat: https://github.com/ruby-bench/ruby-bench-docker/blob/master/README.md#run-ruby-benchmarks-1

File: rails/rails_releases/rails_benchmarks/Dockerfile (hunk 1 above)
Comment (klgilbert) [inline rails/rails_releases/rails_benchmarks/Dockerfile]: That's a mistake. I'll fix that

File: rails/rails_releases/rails_benchmarks/runner (hunk 2 above)
Comment (klgilbert) [inline rails/rails_releases/rails_benchmarks/runner:35]: Updated the README

**Comment by tgxworld** [comment]
Awesome! Thanks! I'll merge this soon as I'll be away for the next few days.
